src/modes/summary/Overview.svelte

```diff
@@ -30,44 +30,18 @@
   $: hospitalTrend = trends[1];
   $: deathTrend = trends[2];
 
-  let minMaxDate = new Date();
+  $: minMaxDate = new Date(Math.min(...[CASES, HOSPITAL_ADMISSION].map((s) => s.timeFrame.max)));
```

Comment:
🤌 👨‍🍳

Reply:
Oh, maybe add a lil comment here to make sure people understand your tight code?